operations/observability/mixins/workspace/rules/central/image-builder.yaml

```diff
@@ -62,3 +62,15 @@ spec:
         expr: |
           kube_deployment_status_replicas_unavailable{deployment="image-builder-mk3", cluster!~"ephemeral.*"} > 0
         for: 10m
+      - alert: GitpodImageBuilderMk3InternalErrors
+        labels:
+          severity: error
```

Comment:
This will alert in Slack. The risk is low even if we flip it to critical (historical data suggests this is rare, especailly for 1h).